spec/spec_helper.rb

```diff
@@ -4,6 +4,8 @@
 require_relative 'support/ruby_model'
 require 'data_magic'
 
+Watir.default_timeout = 5
```

Comment:
I will likely implement something [like this](https://github.com/watir/watir/blob/master/spec/watirspec/support/raise_exception_matchers.rb) for these